hardware_interface/include/adcs.h

```diff
@@ -30,6 +30,143 @@ typedef struct __attribute__((packed)) {
     xyz16 Rate_Sensor_Temp;
 } ADCS_HouseKeeping;
 
+typedef struct __attribute__((packed)) {
+    uint8_t node_type;
+    uint8_t interface_ver;
+    uint8_t major_firm_ver;
+    uint8_t minor_firm_ver;
+    uint16_t runtime_s;
+    uint16_t runtime_ms;
+} ADCS_node_identification;
+
+typedef struct __attribute__((packed)) {
+    uint8_t mcu_reset_cause;
+    uint8_t boot_cause;
+    uint16_t boot_count;
+    uint8_t boot_idx;
+} ADCS_boot_program_stat;
+
+typedef struct __attribute__((packed)) {
+    uint8_t program_idx;
+    uint8_t boot_stat;
+} ADCS_boot_index;
+
+typedef struct __attribute__((packed)) {
+    uint32_t time;
+    uint8_t event_id;
+    uint8_t event_param;
+} ADCS_last_logged_event;
+
+typedef struct __attribute__((packed)) {
+    bool format_busy;
+    bool erase_all_busy;
+} ADCS_SD_format_progress;
```

Comment:
I'm not sure how packed structs with bools would work but probably better not to copy this struct to the packet. I think parsing 0x0101 is better than 0x03.

Reply:
Sure I can separate them